### ISSUE TITLE
when multiple /r/ variants are selected in word quest, only one is shown in trials 11-70

### ISSUE THREAD

**Issue by erinmdoty** [ByunLab/staRt-ionic#271]
When in a word quest with more than one /r/ variant, only words from one of those variants are presented in trial 11-70 or 80.

Example: I selected all /r/ variants for a word quest. In trials 11-80 all the words presented were from the list of consonantal front words.

I tried this multiple times, selecting multiple combinations of /r/ variants, and the same thing happened.

It is important to note that the adaptive difficulty still worked; even though only one /r/ variant was being shown, it moved up in level (harder words, then in phrases, then in sentences) appropriately.

**Comment by erinmdoty** [comment]
Running version 1.0.26 of staRt on iPad 4th generation (AI1458) with iOS 10.3.3

**Comment by starakaj** [comment]
@jonathan-chin have you gotten the chance to look at this?

**Comment by starakaj** [comment]
Can't reproduce on the device either. Erin, can you just verify that this is still an issue with v1.0.31?

**Comment by erinmdoty** [comment]
@starakaj  

I just tried a bunch of different combinations of /r/ variants to see if certain combinations were producing this issue. So far, from what I can tell, if I select "good" after each word, there isn't a problem, but if I select "great" all or most of the time, then the same /r/ variant gets repeated for a long stretch. For example, I selected consonantal front and vocalic front and only selected "great" after each word. The first 30 words were random, words 31-50 were all cons. front and words 50-100 were all vocalic front. I'll try a few more combinations of variants and combinations of responses (great/good/try again) to see if there are any other patterns to when randomization happens and when it doesn't.
